### ISSUE TITLE
Installation error - emr.R emr.add.instance.groups function

### ISSUE THREAD

**Issue by BayanHammami** [armstrtw/Rawscli#3]
I got this error while trying to use devtools::install_github. 

Looks like the emr.add.instance.groups function didn't generate correctly and there is still '<value>' in the function which I'm guessing comes from the function template used.

This looks very interesting and something that I wouldn't mind helping out with if you're busy.

`Error in parse(outFile) : 
  /private/var/folders/hd/rrxlsj6142568hzc5m537fnw0000gn/T/RtmprcLkiz/devtools16f0190734e4/armstrtw-Rawscli-507c241/R/emr.R:1:70: unexpected '<'
1: emr.add.instance.groups <- function (cluster.id=NULL,instance.groups <
                                                                         ^
ERROR: unable to collate and parse R files for package ‘Rawscli’
* removing ‘/Library/Frameworks/R.framework/Versions/3.3/Resources/library/Rawscli’
Installation failed: Command failed (1)`

**Comment by armstrtw** [comment]
yep, well. this package never really got fully developed.  if you want to clone and give it a go, that would be great.  some of the documentation files were a little hard to parse, as the argument format was not consistent (just from memory, it's been a while since I've looked at it).

**Comment by BayanHammami** [comment]
What do you think of this as an potential solution to ensure the package can be installed? It doesn't solve the parsing issue but at least it cat's an empty string into the commands file rather than a function that can't be parsed.

```
fun_str <- "x <- function(a){

print(a)
<random garbage causing an error>
}"

new_fun <- tryCatch(
  {
    eval(parse(text=fun_str))
    fun_str
  },
  error = function(e)
  ""
)

new_fun
```

This can go in generate.function.R just before the last line. If you think its an okay solution, Ill make a pull request.
